src/pegmatite/pattern/repeat.cr

```diff
@@ -16,9 +16,9 @@ module Pegmatite
       @child.inspect(io)
       io << ".repeat("
       @min.inspect(io)
-      if max = @max
+      if @max < Int32::MAX
```

Comment:
Not sure if it makes more sense to do this, or just to let it say `repeat(0 - 2147483647)`. I think this is probably cleaner.

Reply:
Or I could just let `@max` be nilable, and further down do `if count < @min || ((max = @max) && count > max)`